packages/react-components/src/theme-provider/src/ThemeContext.ts

```diff
@@ -1,13 +1,13 @@
 import { createContext, useContext } from "react";
-import type { ColorScheme, Theme } from "./ThemeProvider";
+import type { ColorScheme, ColorSchemeOrSystem, Theme } from "./ThemeProvider";
 
 export interface ThemeContextType {
-    theme?: Theme;
-    colorScheme?: ColorScheme;
-    setColorScheme?: (newColorScheme: ColorScheme) => void;
+    theme: Theme;
+    colorScheme: ColorScheme;
+    setColorScheme: (newColorScheme: ColorSchemeOrSystem) => void;
 }
 
-export const ThemeContext = createContext<ThemeContextType>({});
+export const ThemeContext = createContext<ThemeContextType>({} as ThemeContextType);
```

Comment:
did you mean specifying default value here? if so, will it look awkward for setColorScheme? but i'm ok with giving initial values here

Reply:
No, sorry I though it was the component. Still the `<ThemeProvider>` component should define a define value for `colorScheme`.

Reply:
I was confused, my bad.